starter-code/server.js

```diff
@@ -71,20 +71,24 @@ app.post('/articles', function(request, response) {
 });
 
 app.put('/articles/:id', function(request, response) {
-  // TODO: Write a SQL query to update an author record. Remember that our articles now have
+  // done: Write a SQL query to update an author record. Remember that our articles now have
   // an author_id property, so we can reference it from the request.body.
-  // TODO: Add the required values from the request as data for the SQL query to interpolate
+  // done: Add the required values from the request as data for the SQL query to interpolate
   client.query(
-    ``,
-    []
+    `UPDATE authors
+    SET author=$1, "authorUrl"=$2
+    WHERE author_id=$3`,
+    [request.body.author, request.body.authorUrl, request.body.author_id]
   )
   .then(function() {
-    // TODO: Write a SQL query to update an article record. Keep in mind that article records
+    // done: Write a SQL query to update an article record. Keep in mind that article records
     // now have an author_id, in addition to title, category, publishedOn, and body.
-    // TODO: Add the required values from the request as data for the SQL query to interpolate
+    // done: Add the required values from the request as data for the SQL query to interpolate
     client.query(
-      ``,
-      []
+      `UPDATE articles
+      SET title=$1, category=$2, "publishedOn"=$3, body=$4, author_id=$5
+      WHERE article_id = $6`,
+      [request.body.title, request.body.category, request.body.publishedOn, request.body.body, request.body.author_id, request.body.article_id]
```

Comment:
You get `article_id` from line 73 `app.put('/articles/:id', function(request, response) {`
which is `request.params.id`